elyra/pipeline/kfp/processor_kfp.py

```diff
@@ -495,7 +497,32 @@ def _cc_pipeline(
 
                 operation_artifact_archive = self._get_dependency_archive_name(operation)
 
-                self.log.debug(f"Creating pipeline component:\n {operation} archive : {operation_artifact_archive}")
+                self.log.debug(
+                    f"Creating pipeline component archive '{operation_artifact_archive}' for operation '{operation}'"
+                )
+
+                volume_mounts = operation.component_params.get(MOUNTED_VOLUMES)
+                if operation.component_params.get(MOUNTED_VOLUMES):
```

Comment:
Could you please move this to a method on `RuntimePipelineProcessor` since #2680 has the exact same code for airflow?  Since it appears that `None` is the indicator for there being no volume mounts configured,  possible signature would be:
```python
def get_volume_mounts(self, mounted_volumes: Optional[List[str]]) -> Optional[Dict[str,str]]:
```
When #2679 lands I think it could become...
```python
def get_volume_mounts(self, mounted_volumes: Optional[KeyValueList]) -> Optional[Dict[str,str]]:
```